### PR TITLE
Add global camera to allow setting page clearColor

### PR DESCRIPTION
**Pull request by paulmasson**
Improvements to documentation and WebGL multiple elements example
#7936

### CODE REVIEW

File: docs/api/renderers/WebGLRenderer.html [hunk 1]
```diff
@@ -169,7 +169,7 @@
 		Return a [page:Boolean] true if the context supports vertex textures.
 		</div>
 
-		<h3>[method:null setSize]( [page:Integer width], [page:Integer height] )</h3>
+		<h3>[method:null setSize]( [page:Integer width], [page:Integer height], [page:Boolean updateStyle] )</h3>
```
Comment (WestLangley) [inline docs/api/renderers/WebGLRenderer.html]: What does `updateStyle` do?
This change should be in a separate PR, actually.

**Comment by WestLangley** [comment]
Thank you for your effort, but with all due respect, what you are doing here by adding a "global camera" is a hack. We have to identify the cause of the problem and fix it.

In the `render()` method of the example, try replacing
```javascript
renderer.setScissorTest( false );
```
with
```javascript
renderer.setScissorTest( false );
renderer.state.setScissorTest( false );
```

If you agree that this solves the problem, then we have a bug in the renderer.

**Comment by mrdoob** [comment]
Fixed. See #7935.
